sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

```diff
@@ -311,7 +311,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 2.4. Create MLTable data input\n",
+    "## 2.4. Convert annotation file from COCO to JSONL\n",
+    "If you want to try with a dataset in COCO format, the scripts below shows how to convert it to `jsonl` format. The file \"odFridgeObjects_coco.json\" consists of annotation information for the `odFridgeObjects` dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate jsonl file from coco file\n",
+    "!python coco2jsonl.py \\\n",
+    "--input_coco_file_path \"./odFridgeObjects_coco.json\" \\\n",
+    "--output_dir \"./\" --output_file_name \"odFridgeObjects_from_coco.jsonl\" \\\n",
+    "--task_type \"ObjectDetection\" \\\n",
+    "--base_url uri_folder_data_asset.path"
```

Comment:
Shouldn't this be --base_url uri_folder_data_asset.path + "images/" ?
Could you try submitting a run with MLTable referring to the generated jsonl and see if it works.
Similar to the visualization, this is not checked in the GH action. So just to make sure it is working.

Reply:
Thanks for noticing it Phani. It should be appended with images/. I made changes and also tested with job by making MLtable point to odFridgeObjects_from_coco.jsonl.

Reply:
Is $base_url required ?